### ISSUE TITLE
[Login] Truncated text and button labels on login screens with larger text sizes

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-ios#14974]
**Describe the bug**
When the device is set to larger text sizes with dynamic text, some of the text and button labels are truncated.

**To Reproduce**
1. Set the device text to a larger size, e.g. 190%.
2. Open the app to the login screen (log out if needed first).
3. Enter a non-WordPress URL and notice that the error screen text is truncated and the button labels are not shown.
4. Enter the URL for a Jetpack-connected store, and then enter an email address that is not registered at WordPress.com. Notice the email error screen has truncated/not visible button labels.
5. Log in to a non-WooCommerce, Jetpack-connected site and notice that the button labels on the WordPress.com email screen overflow the available space.
6. On the password screen, notice the magic link button label is truncated.
7. On the non-WooCommerce site error screen, notice the button label is truncated.
8. Tap "See Connected Stores" and notice the login button label is truncated.
9. Log in to an account with 2FA and notice the "Text me a code" button label is truncated.

**Screenshots**

Step 3|Step 4|Step 5|Step 6|Step 7|Step 8|Step 9
-|-|-|-|-|-|-
![Image](https://github.com/user-attachments/assets/008c6cd9-a5d6-44ff-a8f3-282706263763)|![Image](https://github.com/user-attachments/assets/6edb12ca-d56a-44ab-a95c-835ed552f23c)|![Image](https://github.com/user-attachments/assets/3a685588-c7c9-4612-ad1d-5f2e7fe54421)|![Image](https://github.com/user-attachments/assets/424d6811-177a-47b5-bc86-d4c69b2e1b19)|![Image](https://github.com/user-attachments/assets/0fc38039-2b4b-4e2c-a0aa-20e104fd9e77)|![Image](https://github.com/user-attachments/assets/fc4ee8d4-3b7f-4fdd-8944-88903a52cb94)|![Image](https://github.com/user-attachments/assets/816e960d-cbb4-401f-a38a-820ba2a73a60)

**Expected behavior**
The screens and buttons should adjust to display the full text at larger text sizes.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
